includes/views/advanced-status.php

```diff
@@ -55,10 +57,16 @@
 	),
 	'GMagick or IMagick' => array(
 		'required' => __( 'Better with transparent PNG images', 'woocommerce-pdf-invoices-packing-slips' ),
-		'value'    => null,
-		'result'   => extension_loaded( 'gmagick' ) || extension_loaded( 'imagick' ),
+		'value'    => $imagick ? 'IMagick ' . phpversion( 'imagick' ) : ( $gmagick ? 'GMagick ' . phpversion( 'gmagick' ) : '' ),
```

Comment:
It seems in other cases we default to `null` but here to an empty string. Is that on purpose?

Reply:
I'm not seeing that. Are you in the latest changes of the branch?